cocos/2d/CCParticleSystem.cpp

```diff
@@ -846,6 +848,8 @@ void ParticleSystem::update(float dt)
         _emitCounter -= rate * emitCount;
```

Comment:
I won't know how scheduler been designed, if it won't accept negative value this bug won't occur, but now it can accept any value